lib/email/sender.rb

```diff
@@ -253,7 +253,7 @@ def send
       if post.present?
         @stripped_secure_upload_shas = style.stripped_upload_sha_map.values
         add_attachments(post)
-      elsif @email_type == :digest
+      elsif @email_type.to_s == "digest"
```

Comment:
Using `#to_s` here for extra robustness.

Reply:
Might be worth doing `@email_type = email_type.to_s` in the constructor, since we are also doing `@email_type.to_s` elsewhere in the code:

https://github.com/discourse/discourse/blob/fb41e47ed0229af2f30769bd03440d779793f5fa/lib/email/sender.rb#L30-L35

Reply:
Going to hold off on this for now, as we're triggering a Discourse event which includes the e-mail type, and figuring out all the consumers will take some work.